design/design_resort.md

```diff
@@ -0,0 +1,62 @@
+
+- The chapter starts with introductory levels that introduce to main mechanics.
+- Then the mess levels tests those mechanics.
+- The roof levels test the skill of player in those mechanics along with a boss fight.
+
+## Key Levels from Celeste (2018)
+
+All the images are from [Berrycamp](https://berrycamp.github.io). Check out the maps on the website for the full levels.
+
+### Introductory Levels
+
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/s1.png" alt="s1" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/s3.png" alt="s3" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/00-a.png" alt="00-a" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/03-a.png" alt="03-a" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/05-a.png" alt="05-a" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/07-a.png" alt="07-a" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/07-b.png" alt="07-b" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/08-c.png" alt="08-c" />
+
+### Mess Levels
+
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/11-x.png" alt="11-x" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/11-y.png" alt="11-y" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/10-x.png" alt="10-x" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/10-d.png" alt="10-d" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/11-b.png" alt="11-b" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/13-b.png" alt="13-b" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/13-a.png" alt="13-a" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/12-x.png" alt="12-x" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/09-d.png" alt="09-d" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/06-d.png" alt="06-d" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/02-d.png" alt="02-d" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/00-d.png" alt="00-d" />
+
+### Roof Levels
+
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof01.png" alt="roof01" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof03.png" alt="roof03" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof04.png" alt="roof04" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof06.png" alt="roof06" />
+<img src="https://berrycamp.github.io/img/celeste/previews/resort/a/roof07.png" alt="roof07" />
+
+# Chapter Arc
+
+It should be said that first 4-6 levels are single screen ones and scrolling is gradually added later in the chapter. The most scrolling would be found in mess levels and roof levels.
+
```

Comment:
Should probably discuss the Oshiro boss fight taking place in the roof levels here